rdmo/core/tests/test_project_status.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from django.core.management import call_command
+
+
+def test_makemigrations_has_no_changes(db, capsys):
```

Comment:
Should this run only in CI?
We could add a pytest marker, that is not run be default.

Reply:
I think its fine to have that locally.